modules/services/sxhkd.nix

```
@@ -0,0 +1,85 @@
+{config, lib, pkgs, ...}:
+
+with lib;
+
+let
+
+  cfg = config.services.sxhkd;
+
+  hotkeybindingsStr = concatStringsSep "\n" (
+    mapAttrsToList (hotkey: command:
+      optionalString (command != null) ''
+        ${hotkey}
+          ${command}
+      ''
+    )
+    cfg.hotkeybindings
+  );
+
+in
+
+{
+  options.services.sxhkd = {
+    enable = mkEnableOption "Simple X hotkey daemon";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.sxhkd;
+      defaultText = literalExample "pkgs.sxhkd";
+      description = "sxhkd package to install.";
+      example =  literalExample "pkgs.sxhkd";
+    };
+
+    hotkeybindings = mkOption {
```

Can rename the option to match other modules that define key bindings:
```suggestion
    keybindings = mkOption {
```